### DISCUSSION TITLE
Files are missing when publishing asp mvc core c# visual studio .net7

### DISCUSSION THREAD

**Discussion by EnesGundogdu13** [dotnet/aspnetcore#58827] [Q&A]
Hello, I am developing a website on asp mvc core c# .net7 via visual studio and I have published my web page, but the files that should be in it do not come. The incoming files are as follows;

wwroot Folder

appsettings.development.json

appsettings.json

Microsoft.EntityFrameworkCore.Abstractions.dll

Microsoft.EntityFrameworkCore.dll

Microsoft.EntityFrameworkCore.Relational.dll

MySqlConnector.dll

Pomelo.EntityFrameworkCore.MySql.dll

Portegu.dll

Portegu.exe

Portegu.pdb

Portegu.runtimeconfig.json

web.config

X.pagedList.dll

X.pagedList.Mvc.Core.dll

These files are available. None of my view files are coming and when I publish my site in this way, my web page does not work. I select the following options when publishing my project to the folder;

Configuration: Release

Target Framework: net7.0

Deployment Mode: Frame dependent

Target Runtime: Portable

I have tried all other settings except these, but the files are always missing. How can I solve this problem? Did I convey my problem in detail?

**Comment by martincostello** [comment]
.NET 7 is [no longer supported](https://devblogs.microsoft.com/dotnet/dotnet-7-end-of-support/) - do you have the same problem after upgrading to .NET 8?

**Reply by EnesGundogdu13** [reply]
I tried to create and publish a project with .net8 version but I had the same problem.
